docs/GettingStarted.rst

```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
 you start exploring the Social Intranet by suggesting you where to go
 and what you should do first via the following links:
 
--  :ref:`Add a profile picture <Change-your-avatar>`
-
--  :ref:`Connect to coworkers <Send-Connection-Request>`
+-  :ref:`Set your profile picture <Change-your-avatar>`
 
 -  :ref:`Join a space <Join-space>`
 
--  :ref:`Post an activity <Share-AS>`
+-  :ref:`Add Connections <Send-Connection-Request>`
 
--  :ref:`Upload a document <Uploading-files>`
+-  :ref:`Post an activity <Share-AS>`
 
-|image43|
+|DW-1|
 
 Clicking each link will direct you to the related page to do the action.
 
 After each action is performed, it will be remarked as completed with a
-strike-through even though it is not performed via this application.
-Also, the completion percentage is updated on the percentage bar.
+check-mark even though it is not performed via this application.
 
-When all the actions are performed, the completion percentage will be
-100%. You can remove this application from the homepage by clicking
-Close or by hovering your cursor over the application header, and click |image44|.
+When all the actions are performed, you can remove this widget from the Stream Page by clicking
+Close on the header|DW-2|.
 
-|image45|
+    .. note:: - You can’t get the Getting Started widget back when it is removed.
 
-    .. note:: - You cannot get the Getting Started application back when it is removed. 
-        - The "Upload a document" action is considered as completed only when a document has been uploaded in your Personal Documents drive.
-    
-    
-.. _CalendarApp:
+.. _WelcomeBack:
 
-Calendar
----------
+Welcome Back
+-------------
 
-The Calendar application displays some calendars and all of their events
-and tasks scheduled in the Calendar applications of Intranet and spaces.
-When going to the homepage, you will see events with their start and end
-date and tasks of Today. You can also see the events and tasks of the
-previous/next day by clicking the previous/next arrow respectively.
+The welcome back widget is an block that welcomes you with some information about your spaces, connections, 
+your points obtained in the current week and your weekly rank.
 
-|image46|
+|DW-3|
 
--  To view details of an event/task directly in the Calendar
-   application, click your desired event/task.
+- You will see your avatar and your name just after the welcome Back label.
```

Comment:
```suggestion
- You can see your profile picture and your name just after the welcome Back label.
```

I hate the term avatar
turn this sentence to present tense. Future makes no sense when describing an interface.